rust/tvm-sys/build.rs

```diff
@@ -19,22 +19,39 @@
 
 extern crate bindgen;
 
+use std::env;
 use std::path::PathBuf;
 
-use std::env;
+use anyhow::{Context, Result};
 
-fn main() {
+fn main() -> Result<()> {
     let tvm_home = option_env!("TVM_HOME").map(str::to_string).unwrap_or({
```

Comment:
(Not really related to `anyhow`)
`unwrap_or` will do the `CARGO_MANIFEST_DIR` work even if `TVM_HOME` is set (because `unwrap_or` is strict). `.unwrap_or_else(|| { ... ` will skip that work, if that's the desired behavior.

Reply:
These are from the "old" bindings so I don't even know who wrote all this code originally, but this seems like better behavior.